### PR TITLE
Make the README file appear on the PyPI project page.

### PR DESCRIPTION
**Pull request by obi1kenobi**
H/t @ColCarroll, inspired by https://github.com/ColCarroll/callisto/blob/master/setup.py

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -40,33 +40,40 @@ def find_name():
...
-          'Programming Language :: Python :: 3',
-          'Programming Language :: Python :: 3.5',
-          'Programming Language :: Python :: 3.6',
-      ],
-      keywords='graphql database compiler orientdb',
-      python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*',
-      )
+def find_long_description():
+    """Return the content of the README.md file."""
+    return read_file('../README.md')
```
Comment (jmeulemans) [inline setup.py]: It looks like we run the `make_new_release.sh` from the top-level of the repo, so wouldn't this be `./README.md`?

Comment (obi1kenobi) [reply setup.py]: I don't think so -- `read_file()` attempts to read a "package" file from inside the `graphql_compiler` directory of the repo ([line 19](https://github.com/kensho-technologies/graphql-compiler/pull/133/files#diff-2eeaed663bd0d25b7e608891384b7298R19)), so we have to go up a directory from there. That's why `find_name` and `find_version` can read `__init__.py` directly.

Comment (jmeulemans) [reply setup.py]: Oh yes you're right, I missed that. Looks good then.

**Comment by ColCarroll** [comment]
Needs a new release before it shows up, right?

**Comment by obi1kenobi** [comment]
Right, it won't be visible on pypi until the next release.

**Pull request merged**